db/schema.rb

```diff
@@ -308,7 +308,6 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
this should stay in

Reply:
uhh. sorry. my local env is messing with me.. :/